code/game/objects/items/weapons/defib.dm

```diff
@@ -342,6 +342,11 @@
 				user.visible_message("<span class='notice'>[user] places [src] on [M.name]'s chest.</span>", "<span class='warning'>You place [src] on [M.name]'s chest.</span>")
 				playsound(get_turf(src), 'sound/machines/defib_charge.ogg', 50, 0)
 				var/mob/dead/observer/ghost = H.get_ghost()
+				if(!ghost.client)
+					// In case the ghost's not getting deleted for some reason
+					log_to_dd("Ghost of name [ghost.name] is bound to [H.real_name], but lacks a client. Deleting ghost.")
+					qdel(ghost)
+					ghost = null
```

Comment:
So if someone logs out, temporarily, then back in, their ghost will be gone?

Are you sure this is safe and won't have...well, side effects?